client/components/jetpack/daily-backup-status/status-card/backup-last-failed.tsx

```diff
@@ -0,0 +1,47 @@
+import { ExternalLink } from '@wordpress/components';
+import { useCallback } from '@wordpress/element';
+import { Icon, info } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import type { Moment } from 'moment';
+
+type Props = {
+	baseBackupDate: Moment;
+	eventsCount: number;
+	selectedBackupDate: Moment;
+	learnMoreUrl?: string;
+};
+
+export const BackupLastFailed: FunctionComponent< Props > = () => {
```

Comment:
I think the Props type is not needed as you are not receiving/using any prop in this component

Reply:
Copy/paste issue. Thanks!